Add taskschd

hcs64 opened the pull request.

agashlin committed:
Add taskschd

retep998 commented on src/um/taskschd.rs:
```diff
@@ -0,0 +1,1158 @@
+        pName: *mut BSTR,
+    ) -> HRESULT,
+    fn Stop(
+    ) -> HRESULT,
```
If the function has no parameters, it should be a single line.

hcs64 replied:
done, thanks for the review!

agashlin committed:
Update taskschd

- zero-parameter methods on single lines
- use ctypes for long and short LONG and SHORT where present in the header
- some additional interfaces from a newer DLL (10.0.17763.1)

retep998 commented on src/um/taskschd.rs:
```diff
@@ -0,0 +1,1237 @@
+        ppEnum: *mut LPUNKNOWN,
+    ) -> HRESULT,
+}}
+
```
Evil blank line.

retep998 commented on src/um/taskschd.rs:
```diff
@@ -0,0 +1,1237 @@
+        pStop: VARIANT_BOOL,
+    ) -> HRESULT,
+}}
+
```
Evil blank line.

agashlin committed:
Fix blank lines
